<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubjectInsertForm</class>
 <widget class="QWidget" name="SubjectInsertForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>451</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>791</width>
    <height>451</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>791</width>
    <height>451</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Предмет, добавление</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#SubjectInsertForm {

	background-color: #284B63;
}</string>
  </property>
  <widget class="QLabel" name="headText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>771</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#headText {

	background-color: #FFC300;
	color: #1A3344;
	border: 5px solid #1A3344;
}</string>
   </property>
   <property name="text">
    <string>Добавление нового предмета</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="IDLine">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#IDLine {

	border: 5px solid #1A3344;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="text1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#text1 {

	background-color: #284B63;
	color: #FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>Идентификационный номер*</string>
   </property>
  </widget>
  <widget class="QLabel" name="text2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#text2 {

	background-color: #284B63;
	color: #FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>Название предмета</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLine">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#nameLine {

	border: 5px solid #1A3344;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="text3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#text3 {

	background-color: #284B63;
	color: #FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>Сокращённое название</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="shortNameLine">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#shortNameLine {

	border: 5px solid #1A3344;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="countLabel">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>210</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#countLabel {

	background-color: #FFFFFF;
	border: 5px solid #1A3344;
}</string>
   </property>
   <property name="text">
    <string>10</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="countSlider">
   <property name="geometry">
    <rect>
     <x>279</x>
     <y>211</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider#countSlider::handle:horizontal {
background: #FFFFFF;
}

QSlider#countSlider::handle:horizontal:hover {
background: #FFC300;
}

QSlider#countSlider::handle:horizontal:pressed {
background: #3976A0;
}</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="text4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#text4 {

	background-color: #284B63;
	color: #FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>Количество часов</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="logText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>771</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit#logText {

	border: 5px solid #1A3344;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="logHead">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>771</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#logHead {

	background-color: #FFC300;
	color: #1A3344;
	border: 5px solid #1A3344;
}</string>
   </property>
   <property name="text">
    <string>Лог ошибок и предупреждений</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="addingButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#addingButton {

	background-color: #284B63;
	color: #FFFFFF;
	border: 5px solid #FFFFFF;
}
QPushButton#addingButton:hover {

	background-color: #FFFFFF;
	color: #1A3344;
	border: 5px solid #1A3344;
}
QPushButton#addingButton:pressed {

	background-color: #DB1921;
	color: #FFFFFF;
	border: 5px solid #1A3344;
}
QPushButton#addingButton:disabled {

	background-color: #284B63;
	color: #4B7089;
	border: 5px solid #4B7089;
}</string>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>390</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#exitButton {

	background-color: #284B63;
	color: #FFFFFF;
	border: 5px solid #FFFFFF;
}
QPushButton#exitButton:hover {

	background-color: #FFFFFF;
	color: #1A3344;
	border: 5px solid #1A3344;
}
QPushButton#exitButton:pressed {

	background-color: #DB1921;
	color: #FFFFFF;
	border: 5px solid #1A3344;
}
</string>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
